V2

jberglund opened the pull request:
Nu funkare!

jberglund committed:
WIP: FormTask

jberglund committed:
Merge branch 'main' into v2

jberglund committed:
It's aliiive!

jberglund committed:
Linting wanted cased title

jberglund committed:
Förbättringar. (:

jberglund committed:
Nu går man till vecko-vyn när man submittar

cbrevik commented on src/api.ts:
```diff
@@ -1,40 +1,63 @@
 }
+
+export function useHarvestWeek(from: string, to: string) {
+  const { data, isLoading } = useFetch(`${BASE_URL}/time_entries?from=${from}&to=${to}`, {
```
Tror det er lurt å bruker `user_id` her også, jeg som manager i Harvest får alle brukere sine time_sentries med dette kallet. Har et eksempel her: https://github.com/varianter/raycast-harvest/commit/44e899f6f116b464e65cf3680732a2f57ac994d4#diff-769911c416ccf8514d8fd941ae0abe8fb5c606ade0c218e22151a5f5f9f3d700R45

cbrevik reviewed (changes requested):
Tror det er lurt å legge inn user id i preferences som nevnt, men ellers sett igjennom og synes det ser bra ut sjø! 

![season-3-my-two-cents-gif-by-the-simpsons](https://github.com/varianter/raycast-harvest/assets/4932625/dcad3f56-dbac-4996-ac48-02e6b7ec99c1)

jberglund committed:
Added user ID as a preference
You can now directly submit a full day from the tasks view
Updated README: how to get started

jberglund committed:
Now needs user ID as a preference
Full day shortcut
Updated README

jberglund committed:
time_entries takes user_id in the json object if needed

jberglund committed:
Schema had an error: should have been number and nullable

cbrevik reviewed (approved):
![great-job-yes-gif-by-hollywood-suite](https://github.com/varianter/raycast-harvest/assets/4932625/c533b633-c8ae-436b-b7c2-cdf58b31a532)
